src/controllers/connectionManager.ts

```diff
@@ -245,12 +245,19 @@ export default class ConnectionManager {
     }
 
     /**
-     * Get the connection view.
+     * Exposed for testing purposes
```

Comment:
Maybe just type cast to any to use the method in tests? Exposing a setter runs into the risk of accidental modifications.